docs/docs/concepts/foundation/transactions.md

```diff
@@ -11,13 +11,49 @@ On this page you'll learn:
 - The Aztec Kernel and its two circuits: private and public, and how they execute function calls
 - The call stacks for private & public functions and how they determine a transaction's completion
 
+## Simple Example of the (Private) Transaction Lifecycle
+
+The transaction lifecycle for an Aztec transaction is fundamentally different from the lifecycle of an Ethereum transaction.
+
+The introduction of the Private eXecution Environment (PXE) provides a safe environment for the execution of sensitive operations, ensuring that decrypted data are not accessible to unauthorized applications. However, the PXE exists client-side on user devices, which creates a different model for imagining a transaction lifecycle. The existence of a sequencing network also introduces some key differences between the Aztec transaction model and the transaction model used for other networks.
+
+The accompanying diagram illustrates the flow of interactions between a user, their wallet, the PXE, the node operators (sequencers / provers), and the L1 chain.
+
+<Image img={require("/img/transaction-lifecycle.png")} />
+
+1. **The user initiates a transaction** – the user decides to privately send 10 DAI to gudcause.eth. After inputting the amount and the receiving address, the user clicks the confirmation button on their wallet.
+
+_The transaction has not been broadcasted to the sequencer network yet._
+
+2. **The wallet and PXE interact** – once the user has confirmed the transfer, the wallet sends an execution request to the PXE, which is running locally on the user device. The PXE executes the transfer method on the DAI token contract on Aztec and computes the state difference based on the user’s intention, which is shared with the wallet to be displayed to the user.
+
+_The transaction has still not been broadcasted to the sequencer network yet._
+
+3. **The wallet asks the user for approval** – the user sees the state difference and can confirm that this matches the intended transaction. This is not a required part of the protocol, but is good wallet UX. The user can then approve the transaction using their existing, selected authorization mechanism (e.g., FaceID, ECDSA signature, etc.).
+
+_The transaction has still not been broadcasted to the sequencer network yet._
+
+4. **The PXE proves correct execution** – now that the user has approved the transfer, the PXE receives the authorization and proves correct execution (via zero-knowledge proofs) of the authorization and the private transfer method. Once the proofs have been generated, the PXE sends the proofs and required inputs (inputs are new note commitments, stored in the [note hash tree](../advanced/data_structures/trees.md#note-hash-tree) and nullifiers stored in the [nullifiers tree](../advanced/data_structures/trees.md#nullifier-tree)) to the sequencer.
```

Comment:
I like how you are actively tried to not use jargon.
Makes me think should we remove the above hyper-links if the goal is a simple overview?

Reply:
I'd prefer to leave the hyperlinks so people that want to dig deeper know where they should go.